command.go

```diff
@@ -783,3 +783,80 @@ func (cmd *ClusterSlotCmd) parseReply(cn *conn) error {
 	cmd.val = v.([]ClusterSlotInfo)
 	return nil
 }
+
+//------------------------------------------------------------------------------
+
+type Location struct {
+	Name string
+	Distance string
```

Comment:
0 would mean at the exact point. If it's a float64 and user didn't ask for distance calculation - it will be 0 but doesn't mean it's a the exact point - it should be undefined instead.

(Geohash does not have this problem, we know 0 is invalid)